wifi_manager/wifi_manager.py

```diff
@@ -931,14 +974,24 @@ def run(self,
                             show debugger content
         :type       debug:  bool, optional
         """
-        self.logger.debug('Run app on {}:{} with debug: {}'.format(host,
-                                                                   port,
-                                                                   debug))
+        self.logger.info('Run app on {}:{} with debug: {}'.format(host,
+                                                                  port,
+                                                                  debug))
+
+        def before_request(req: Request) -> None:
+            gc.collect()
+            gc.threshold(gc.mem_free() // 4 + gc.mem_alloc())
+            free = gc.mem_free()
+            self.logger.info(f"Before request: {req.url}, free mem: {free}")
```

Comment:
I see no benefit in having the info level here ... this might become quite noisy. I would even delete it completely as this function gets called quite often

```suggestion
            self.logger.debug(f"Before request: {req.url}, free mem: {free}")
```

Reply:
Actually, I'd changed a couple of these from `debug` to `info` because I'm still not seeing any of the `debug` messages. I'll change them back. Perhaps you know why `debug` doesn't seem to work?